src/components/FilterPane/FilterPane.jsx

```diff
@@ -102,7 +102,7 @@ const FilterPane = ({ mermaidUserData }) => {
     countriesSelectOnOpen,
     organizationsSelectOnOpen,
     getActiveProjectCount,
-  } = useFilterProjectsContext()
+  } = useContext(FilterProjectsContext)
   const [expandedSections, setExpandedSections] = useState({
```

Comment:
There are a few files with this change. The linter didnt like the custom hook being defined in the context file, so I just used the more modern way to access context directly.